### ISSUE TITLE
GuzzleHttp-issue with Chromium and GDrive bookmark

### ISSUE THREAD

**Issue by Larx** [marcelklehr/floccus#173]
### Software versions

[](url)
* Browser: Version 69.0.3497.100 (openSUSE Build) (64-Bit)
* Nextcloud: 14.0
* Nextcloud Bookmarks app: 0.13.0
* Floccus: 3.0.8

### Steps to reproduce

1. Nothing special: Everything has worked flawlessly for the last weeks.
2. I add the following bookmark: 
https://drive.google.com/drive/folders/1HtRN6OnYgGsPv2zTztcUYa-3c6Bq-fJv
3. Syncing fails with Chromium
4. I remove the bookmark: Everything works again.

### Expected outcome

Chromium should sync its bookmarks.

### Actual outcome

Chromium causes a GuzzleHttp-error with NextCloud (and a POST error in the Chromium app). Firefox and the Android app still work
...

![bildschirmfoto vom 2018-09-23 06-39-09](https://user-images.githubusercontent.com/3582007/45924247-6a0b6300-befb-11e8-88b5-547b2d5ae9ae.png)

**Comment by marcelklehr** [comment]
Did you recently update to v14? Have you had trouble adding any other bookmarks since? Are you able to open the "Apps" admin page without problems? Do you have the app "files_external_gdrive" enabled?

**Comment by Larx** [comment]
Yes. Yes, other links to GDrive, everything else is OK. Yes. Yes.
I had to disable files_external_gdrive for updating, but was able to reenable it afterwards.
